src/app/profile/payment-methods/payment-method/forms/add-payment-method/modal.tpl.html

```diff
@@ -0,0 +1,86 @@
+<div class="modal-content give-modal-content">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+
+        <div class="col-xs-12">
+          <button ng-click="$ctrl.cancel()" type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+          </button>
+          <h4 class="modal-title border-bottom-small">Add Payment Method</h4>
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body">
+    <div class="container-fluid" tabindex="-1" role="dialog">
+      <div class="row">
+        <div class="col-xs-12">
+
+          <div class="alert alert-danger" role="alert" ng-if="$ctrl.error">
+            <p>{{$ctrl.error}}</p>
+          </div>
+
+          <div class="mb_x">
+            <div class="row">
```

Comment:
It might be nice to share some of this html with what the step-2 component has. It might take some refactoring in the checkout component or adding an intermediate component though since there are next step buttons there... I'll think about that more